charts_plugin/src/main/java/com/futureworkshops/mobileworkflow/plugin/charts/view/UINetworkChartPluginStep.kt

```diff
@@ -1,49 +1,56 @@
-package com.futureworkshops.mobileworkflow.plugin.charts.pie.view
+/*
+ * Copyright (c) 2022 FutureWorkshops. All rights reserved.
+ */
 
+package com.futureworkshops.mobileworkflow.plugin.charts.view
 
 import com.futureworkshops.mobileworkflow.backend.views.step.FragmentStep
 import com.futureworkshops.mobileworkflow.backend.views.step.FragmentStepConfiguration
 import com.futureworkshops.mobileworkflow.data.network.task.URLIAsyncTask
 import com.futureworkshops.mobileworkflow.data.network.task.URLMethod
 import com.futureworkshops.mobileworkflow.model.AppServiceResponse
 import com.futureworkshops.mobileworkflow.model.result.AnswerResult
-import com.futureworkshops.mobileworkflow.plugin.charts.pie.step.PieChartItem
+import com.futureworkshops.mobileworkflow.plugin.charts.step.DashboardNetworkChartItem
 import com.futureworkshops.mobileworkflow.services.ServiceBox
-import com.futureworkshops.mobileworkflow.services.localization.LocalizationService.PredefinedText
+import com.futureworkshops.mobileworkflow.services.localization.LocalizationService
 import com.futureworkshops.mobileworkflow.steps.Step
 import com.google.gson.reflect.TypeToken
 
-data class UINetworkPieChartPluginStep(
-    val title: String,
+data class UINetworkChartPluginStep(
     override val id: String,
-    private val url: String
-) : Step {
+    val title: String,
+    val numberOfColumns: Int,
+    val url: String
+): Step {
+
     override fun createView(
         stepResult: AnswerResult?,
         services: ServiceBox,
         appServiceResponse: AppServiceResponse
     ): FragmentStep {
-        val fullUrl = "${appServiceResponse.server?.url}/${url}"
-        
-        val task = URLIAsyncTask<Nothing, List<PieChartItem>>(
+        val fullUrl = "${appServiceResponse.server?.url}${url}"
```

Comment:
@jmcarcedo URLs should be resolved using URL resolver services to be sure to have a valid session values translation. Like, for example, `/chart/{uuid.answer}` being resolved from the session.